aptos-move/block-executor/src/counters.rs

```diff
@@ -148,17 +148,6 @@ pub static TASK_EXECUTE_SECONDS: Lazy<Histogram> = Lazy::new(|| {
     .unwrap()
 });
 
-pub static GET_NEXT_TASK_SECONDS: Lazy<Histogram> = Lazy::new(|| {
-    register_histogram!(
-        // metric name
-        "aptos_execution_get_next_task_seconds",
-        // metric description
-        "The time spent in seconds for getting next task from the scheduler",
-        exponential_buckets(/*start=*/ 1e-6, /*factor=*/ 2.0, /*count=*/ 30).unwrap(),
```

Comment:
How do these exponential buckets work? Is there a counter per bucket (which is bad b/c we can have a huge measurement error); or is it a list per bucket (problem b/c of contention)

Reply:
Yes my understanding that there is a counter for bucket and some approximation algorithm is applied to aggregate across buckets..

Reply:
It's a counter per bucket, mostly used for calculating x%tile. (of course with errors)

There is a separate overall count and sum, so the average is accurate.

Reply:
@grao1991 Wow - that counter actually needs more compute and sync than the loop it's measuring - and in return gives us an approximation.  I'm still curious why we got this counters at the first place - just to make sure we dont do it again.

Reply:
I have no idea, we should never put this counter on a very hot code path, especially in a multi thread context.

Someone needs to implement a per CPU counter if we really need such measurement.

Reply:
@ziaptos - https://github.com/aptos-labs/aptos-core/pull/6956 has some context on why the counters were added. These block-stm performance counters gave us very valuable insights into performance bottlenecks in block-stm code and helped us address various scalability issues. As I mentioned earlier, this bottleneck is not seen in land blocking forge or in the block-stm microbenchmark (both were run in the PR and we verified no regression). These bottlenecks show up when we are running hot on CPU because of high TPS load.